### ISSUE TITLE
use validation set as model selection cirteria (use training loss for DANN as model selection does not make sense)

### ISSUE THREAD

**Issue by smilesun** [marrlab/DomainLab#213]
currently model selection is MSelTrLoss

**Comment by smilesun** [comment]
https://github.com/marrlab/DomainLab/pull/230
